### ISSUE TITLE
Update environmental AI book

### ISSUE THREAD

**Issue by JimCircadian** [alan-turing-institute/IceNet-Project#27]
Ref: https://acocac.github.io/environmental-ai-book/polar/modelling/polar-modelling-icenet.html

Update for daily SIC forecasts

**Comment by JimCircadian** [comment]
Hi @bnubald please can you keep this issue up to date with progress to the WS2.5 work you're undertaking? This has long been an open issue so it's great it'll be addressed!

**Comment by bnubald** [comment]
> Hi @bnubald please can you keep this issue up to date with progress to the WS2.5 work you're undertaking? This has long been an open issue so it's great it'll be addressed!

Will do.

**Comment by bnubald** [comment]
Starting the EDS book notebook idea review process (first stage of submitting).

alan-turing-institute/eds-ideas-transfer#5

**Comment by bnubald** [comment]
Notebook to submit to EDS book being worked in personal repo:

https://github.com/bnubald/icenet-edsbook

This will be transferred over to the `eds-book-gallery` org once approved, so not located in icenet-ai org.

**Comment by bnubald** [comment]
Updates made to EDS book repo.

Can be accessed either here:
https://mybinder.org/v2/gh/bnubald/icenet-edsbook/dev_bnu

Or here (if above takes a while to set up):
https://github.com/bnubald/icenet-edsbook/blob/dev_bnu/notebook.ipynb

Could have gone the route of using pre-trained models (Like in current icenet demonstrator in edsbook based on original paper).

But, opted instead to go show an entire end to end run using the Python library API.

The training and validation date range has been set to be extremely limited (only 3 days), and so has `n_filters_factor=0.05` and `epochs=3` to keep within Binder's resource limits. Because of this the output result isn't great, so at the end of the notebook, an output from a decent result is shown, and the corresponding updates needed to the notebook to get that result.

Additionally, OSI-SAF data and masks are included since ftp access is blocked on Binder (Though the commands to download are still included to demonstrate how to use them - they just skip over the already downloaded files).

The links at the bottom of the notebook are pointing to as of yet unpushed files on, so that's a todo.

Thoughts welcome! @JimCircadian

**Comment by JimCircadian** [comment]
I think this is absolutely amazing Bryn, you're the binder master, making this...

```
Total params: 24723 (96.57 KB)
Trainable params: 24549 (95.89 KB)
Non-trainable params: 174 (696.00 Byte)
```

...fit into a binder for an actual run is totally awesome. I think get it submitted to EDS. The images don't seem to be working but ultimately, it's illustrative and should give enough of an impression to what's possible. Perhaps sticking some mention of the other types of data is important (e.g. what `icenet_data_...` commands there are) but that might be in there already, sorry if I skimmed over it!

**Comment by JimCircadian** [comment]
I wouldn't worry about todo's, the main thing is to get it into the EDS book and then we can keep it as a future effort for someone introducing themselves to the project to extend it. Ultimately, at some point it'll need updating anyway so working with this as a baseline is perfect. Well done!!! 😉

**Comment by bnubald** [comment]
Example results :( - they do vary on each run...:

![image](https://github.com/icenet-ai/icenet-roadmap/assets/55503826/455ac5b5-ceaa-4c5c-be32-c88fe3261c3a)
![image](https://github.com/icenet-ai/icenet-roadmap/assets/55503826/8b5e28a2-dd1a-4fc7-8764-8756ab7f6b58)
![image](https://github.com/icenet-ai/icenet-roadmap/assets/55503826/6c31b230-da4d-4c6b-a94e-925bb99caf71)

But, do demonstrate a 'good' result, and what changes to make to obtain them.

![image](https://github.com/icenet-ai/icenet-roadmap/assets/55503826/47c3f638-c209-4c0f-a78f-63b8813b5f19)

**Comment by JimCircadian** [comment]
Good enough considering the limitations! :P

**Comment by bnubald** [comment]
Started pre-review process of submitting to [EDS Book](https://edsbook.org/welcome.html).

**Comment by bnubald** [comment]
Initial checks passed, will now go through pre-review/review process.

**Comment by bnubald** [comment]
Initiated updates to submission based on review comments.

**Comment by bnubald** [comment]
Submitted responses to reviews:

* Reviewer 1: https://github.com/alan-turing-institute/notebooks-reviews/issues/7
* Reviewer 2: https://github.com/alan-turing-institute/notebooks-reviews/issues/7

and the... oops too large PR covering these changes:
*  https://github.com/eds-book-gallery/67a1e320-7c47-4ea9-8df8-e868326bc90b/pull/6

**Comment by bnubald** [comment]
Completed further review based updates, been notified that it should be moving to the post-print phase.

1) https://github.com/alan-turing-institute/notebooks-reviews/issues/7
2) https://github.com/alan-turing-institute/notebooks-reviews/issues/7
3) https://github.com/alan-turing-institute/notebooks-reviews/issues/7

Refer to phases here:
https://edsbook.org/publishing/guidelines/guidelines-authors#post-print

**Comment by bnubald** [comment]
This is now completed.

Use of the IceNet library as it currently stands is listed as daily version, and Tom's original paper based notebook written by Alejandro is listed as a separate notebook for monthly prediction.

Once the IceNet library is able to replicate the original study, the monthly notebook should be updated to use this library instead.

Location of published notebook:

https://edsbook.org/notebooks/gallery/67a1e320-7c47-4ea9-8df8-e868326bc90b/notebook
